OEL-1283: Add update instructions for paragraphs move (#99)
openeuropa/oe_whitelabel: README.md

```bash
docker-compose exec web ./vendor/bin/phpunit
```

## Upgrade from older versions

### Upgrade to 1.0.0-alpha6

#### Paragraphs migration

Paragraphs-related theming and functionality has been moved from the [OpenEuropa Bootstrap base theme](https://github.com/openeuropa/oe_bootstrap_theme) to [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel).

Special paragraphs fields that were introduced in `oe_bootstrap_theme_paragraphs` are being renamed in `oe_whitelabel_paragraphs`.

> If you have the `oe_paragraphs` module enabled, you should create a `hook_update_N()` or `hook_post_update_NAME()` in a custom module, to enable the `oe_whitelabel_paragraphs` module during deployment.

Reviewer: ```suggestion
If you have the `oe_paragraphs` module enabled, you should create a `hook_post_update_NAME()` in your code, to enable the `oe_whitelabel_paragraphs` module during deployment.
```

```bash
docker-compose exec web ./vendor/bin/phpunit
```

## Upgrade from older versions

### Upgrade from 1.0.0-alpha5 and earlier

> Target release: You can upgrade directly to the version that contains this README.md.

Reviewer: Again, what does this mean?

> #### Paragraphs migration

Reviewer 2: makes more sense to me saying "Upgrading from version x to y" or "Upgrading to version y"

> Paragraphs-related theming and functionality has been moved from `oe_bootstrap_theme` to `oe_whitelabel`.

Reviewer 2: link oe_whitelabel

If you have the `oe_paragraphs` module enabled, you should:
- Create a `hook_update_N()` or `hook_post_update_NAME()` in a custom module, to enable the `oe_whitelabel_paragraphs` module.
- Run `drush updb` in your local installation.
- Run `drush config:export`, to export the changes.
- For deployment, make sure that `drush updb` runs _before_ `drush config:import`.

If you did _not_ have `oe_paragraphs` enabled, but you want to do so now, you should:
- Enable the new `oe_whitelabel_paragraphs` module in your local installation, ideally with `drush en`.
- Export configuration, e.g. via `drush cex`. This should add the module to `core.extension.yml`.

> - For deployment, `drush config:import` will do the job. No update hook needed.

Reviewer: Information NOT needed. We need to write HOW to do the upgrade from previous version. This is general information written in dozens of pages, no need to write it here again.
